expr/plan.go

```diff
@@ -98,8 +98,20 @@ func newplan(e *expr, context Context, stable bool, reqs []Req) (GraphiteFunc, [
 		req := NewReq(e.str, context.from, context.to, context.consol)
 		reqs = append(reqs, req)
 		return NewGet(req), reqs, nil
+	} else if e.etype == etFunc && e.str == "seriesByTag" {
+		expressionStr := "seriesByTag("
```

Comment:
This builds the original function back up. It's possible we could avoid parsing it in the first place?

Reply:
Alternatively, we can allow for passing more information in the Req type, but it needs to remain hashable for use as a map key.